### ISSUE TITLE
[feat] support load plain worker module in another worker

### ISSUE THREAD

**Issue by JounQin** [webpack-contrib/worker-loader#320]
- Operating System: macOS 12
- Node Version: 12.22.1
- NPM Version: 6.14.12
- webpack Version: 5.39.0
- worker-loader Version: 3.0.8

### Feature Proposal

`excludeImportee` option to exclude worker be loaded as webpack WebWorker

### Feature Use Case

If 'b.worker.js' imports 'a.worker.js' to reuse some of its logic, it will just throw errors

![image](https://user-images.githubusercontent.com/8336744/121989716-2bb2e200-cdcf-11eb-8c16-c88a8743728d.png)

https://github.com/JounQin/test/blob/worker-loader/src/b.worker.js

Please check https://github.com/JounQin/test/blob/worker-loader for example.

---

Workaround:

exclude `a.worker.js` from `worker-loader`, and use `worker-loader!./a.worker` instead.

**Comment by alexander-akait** [comment]
@JounQin Why do you use `worker-loader` with webpack v5?

**Comment by JounQin** [comment]
@alexander-akait Oh, sorry, I didn't notice the native worker support in webpack v5, but using `worker-loader` seems still valid and cleaner, isn't it?

**Comment by alexander-akait** [comment]
@JounQin Yep, worker-loader should work, but here some limitations - cache will not work as expected and some features is not available

**Comment by JounQin** [comment]
That's why I'm asking for this feature here, if it's not possible to support it, feel free to close, I'll continue to use my current workaround as above.

**Comment by alexander-akait** [comment]
@JounQin There is solution for webpack v4 https://github.com/GoogleChromeLabs/worker-plugin, it should be better than this loader (also it will be deprecated in near future)
